tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "incremental": true,
     "importHelpers": false,
-    "target": "ES2018",
+    "target": "ES2015",
```

Comment:
why do we need to change target?

Reply:
There was an `ERR_REQUIRE_ESM` error. I forgot undo that change. Thank you